Song_sort_Genre.php

```diff
@@ -1,39 +1,70 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
-		<title>Layout1Musicplay</title>
+		<title>Sort By Genre</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
+		<!--Connected to css -->
 		<link rel ="stylesheet" href = "test12.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
-		<div class="img_1">
```

Comment:
21/9: Delete the banner because in this layout I decided to make the nav bar to be at the top because I released that the nav bar was importation then the banner So I will delete it for now and apply it latter.